### PR TITLE
Hotfix/remove nginx server #40

### LINKED ISSUE #40: remove nginx, use only envoy
I think, it is possible to remove nginx from docker compose service and go for only envoy, no need to have them both, they have some common  points however envoy's perspective is different.  Although it is different, I think, it can be used for both handling gRPC and HTTP/1 requests thanks to [envoy filters](https://www.envoyproxy.io/docs/envoy/latest/configuration/http/http_filters/http_filters).

What do you think about this @gianmarcomennecozzi ?

### LINKED ISSUE #41: add new set of features
- Timeline  (which represents availability of server in time) respect to booking functions which are introduced in haaukins-store.  
- List events based on request [status].

### PR DESCRIPTION
**Pull request by mrtrkmn**
~- nginx removed~ (Was removed however `http-server` is unable to route between pages on `./dist`)
- envoy updated 

~- routing issue is fixed,  when you type **admin.haaukins.com:8003** _**without**_ **HTTPS** notation it will **NOT** have problems, by returning "BAD REQUEST"~ (did not fixed )

Note: checked, it is running.

### CODE REVIEW

**Comment by mrtrkmn** [comment]
I have removed nginx and serve through http-server however it was not able to route pages on webclient that's why I had to bring back nginx again. Envoy currently unable to serve such a `./dist` file which is output of `npm build`, with grpc server, I noted down things on dockerfile and envoy.yaml, they do not have any harm that's why I think they can stay. 

Although I am unable to remove nginx, I refactored some parts.

File: envoy/envoy.yaml [hunk 1]
```diff
@@ -42,18 +46,39 @@ static_resources:
...
+      http2_protocol_options: {}
+      load_assignment:
+        cluster_name: application_frontend
+        endpoints:
+        - lb_endpoints:
+          - endpoint:
+               address:
+                 socket_address:
+                   address: 127.0.0.1
+                   port_value: 8003
```
Comment (gianmarcomennecozzi) [inline envoy/envoy.yaml:79]: Do we need this code ?

**Review by gianmarcomennecozzi — COMMENTED** [review]
I have testes it in local removing the TLS part on the docker-compose and its working, i am not sure if we really needed some modification you made but looking the home page
Screenshot from 2020-07-06 10-00-17

for sure there is something to fix on haaukins and maybe in the haaukins store as well

File: envoy/envoy.yaml (hunk 1 above)
Comment (mrtrkmn) [reply envoy/envoy.yaml:79]: Hmm, well, I kept it in purpose, to keep in mind that one day envoy will be used only. However, it can be commented.

**Comment by mrtrkmn** [comment]
Related changes added on both haaukins and store, I think, webclient should have at least 3 different buttons(Suspended, Booked, Closed, (Running could be default case)) or somehow filtering section which users can filter out events and see them in more clear way, as default, they can see only running events however if they wish to see other events as well then they should either use filtering functionality or using buttons. I think, that would be enough.  Therefore, for this pull request, there is no need to change. @gianmarcomennecozzi

**Comment by mrtrkmn** [comment]
That one was already added to "to do list"  > https://github.com/aau-network-security/haaukins-webclient/issues/41

**Pull request merged**